helm_deploy/dps-data-compliance/templates/_envs.tpl

```diff
@@ -82,6 +82,16 @@ env:
     value: "{{ .Values.env.OFFENDER_DELETION_WINDOW_LENGTH }}"
   {{- end }}
 
+  {{- if .Values.env.OFFENDER_RETENTION_SQL_DATA_DUPLICATE_CHECK_ENABLED }}
```

Comment:
Do we need all these `if` so it is optional to supply the env value, e.g fallback to the default value?

Reply:
Yes, I seem to think I had problems early on where if I didn't do this, because the environment variable was defined (even though with an empty value) I couldn't fall back to a default